src/licensedcode/data/licenses/ncsa-httpd-1995.LICENSE

```diff
@@ -0,0 +1,44 @@
+---
+key: ncsa-httpd-1995
+short_name: NCSA HTTPd License 1995
+name: NCSA HTTPd License 1995
+category: Proprietary Free
+owner: NCSA - University of Illinois
+homepage_url: https://web.archive.org/web/19970303200757/http://hoohoo.ncsa.uiuc.edu/docs-1.5/COPYRIGHT.html
+spdx_license_key: LicenseRef-scancode-ncsa-httpd-1995
+other_urls:
+    - https://lists.debian.org/debian-mentors/2023/06/msg00060.html
+ignorable_copyrights:
+    - Copyright (c) 1995, Board of Trustees of the University of Illinois
+    - copyrighted by The Board of Trustees of the University of Illinois
+ignorable_holders:
+    - Board of Trustees of the University of Illinois
+    - The Board of Trustees of the University of Illinois
+ignorable_emails:
+    - httpd@ncsa.uiuc.edu
+---
+
+NCSA HTTPd Server
```

Comment:
Why do we need this header and copyright in the license text? IMHO this should not be part of it. @DennisClark ?

Reply:
The license text is an exact copy of the text at
https://lists.debian.org/debian-mentors/2023/06/msg00060.html 
and
https://web.archive.org/web/19970303200757/http://hoohoo.ncsa.uiuc.edu/docs-1.5/COPYRIGHT.html

but if you want to remove the header text that's ok with me.   However, please note that the word "COPYRIGHT" is part of the original ncsa URL, and the copyright statement on that page is very prominently formatted.  So I think we can keep it but I don't feel strongly about it.